execution-core/CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added `groth16` module allowing for proof generation and verification
 
+### Changed 
+
+- Moved the `gas_price` check back from the `econ_protocol` back to the transfer contract
```

Comment:
The economic protocol was never part of execution-core so the check didn't come back in that sense.
```suggestion
- Add check for minimum gas when creating transactions
```